README.md

```diff
@@ -159,6 +159,30 @@ installation and massively reducing the install time on CI.
 When using the [Nix](https://nixos.org/nix) package manager, the `nixpkgs.cargo-tarpaulin` package can be used.
 This ensures that tarpaulin will be built with the same rust version as the rest of your packages.
 
+### Environment Vars
+
+Tarpaulin sets the below environment vars under test:
+
+- **RUST_BACKTRACE**      - _When --verbose flag is used_
+- **CARGO_MANIFEST_DIR**  - _Path to Cargo.toml From --root | --manifest-path or guessed from the current or parent directory_
+- **CARGO_PKG_NAME**      - _From Cargo.toml_
+- **CARGO_PKG_AUTHORS**   - _From Cargo.toml_
+- **CARGO_PKG_VERSION**   - _From Cargo.toml_
+- **LLVM_PROFILE_FILE**   - _Used for LLVM coverage_
+
+**Cargo Limitations**
+
+Since tarpaulin is a custom Cargo sub-command, the regular Cargo environment has been constructed by tarpaulin.
+
+In order for tarpaulin to construct the Cargo environment correctly, tarpaulin needs to find Cargo.toml by either:
+- Using *--root* or *--manifest-path* or
+- By invoking cargo from the current working directory holding the Cargo.toml manifest or
+- By invoking cargo from a child directory under the parent holding the Cargo.toml
```

Comment:
I think something like "By invoking cargo from a sub-directory within the project" would be nicer phrasing. Also for these points if you capitalise cargo to make it consistent with the other usages